packages/meta/src/validatordef/generator/ValidationUtils.ts

```diff
@@ -3,15 +3,15 @@ import {
     FreMetaPrimitiveProperty,
     FreMetaPrimitiveType,
 } from "../../languagedef/metalanguage/index.js";
-import { FreComparator } from '../../validatordef/metalanguage/index.js';
+import { FreComparator } from '../metalanguage/index.js';
 
 
 export class ValidationUtils {
     public static findLocationDescription(concept: FreMetaClassifier | undefined, paramName: string): string {
         let nameProp: FreMetaPrimitiveProperty | undefined = concept
             ?.allPrimProperties()
             .find((prop) => prop.name === "name");
-        if (!!!nameProp) {
+        if (!nameProp) {
```

Comment:
`if (nameProp === undefined) {` is more clear, insytead of the ! (not) to check for undefined